godtools/App/Flows/FlowStep.swift

```diff
@@ -18,7 +18,8 @@ enum FlowStep {
     case showMenu
     case showLanguageSettings
     case showOnboardingTutorial(animated: Bool)
-    case dismissOnboardingTutorial
+    case dismissOnboardingTutorial(dismissOnboardingTutorialType: OnboardingQuickStartView.DismissOnboardingTutorialType?)
```

Comment:
I like the idea here of providing this dismiss type.  It fits well with some places in the app where I've been starting to implement a flow completed state.  See ```LessonFlowCompletedState``` and ```TractFlowCompletedState```.  I'd like to do the same here with our OnboardingFlow and have an ```OnboardingFlowCompletedState``` which will contain those cases you created in ```DismissOnboardingTutorialType```.